tests/unit/lms/services/email_preferences_test.py

```diff
@@ -11,16 +11,17 @@
     factory,
 )
 from lms.services.exceptions import ExpiredJWTError, InvalidJWTError
+from tests import factories
 
 
 class TestEmailPreferencesService:
     def test_unsubscribe_url(self, svc, jwt_service):
         jwt_service.encode_with_secret.return_value = "TOKEN"
 
-        url = svc.unsubscribe_url(sentinel.email, sentinel.tag)
+        url = svc.unsubscribe_url(sentinel.h_userid, sentinel.tag)
```

Comment:
Just correcting a small mistake here: `unsubscribe_url()`'s first argument is `h_userid` not `email`.